lib/utils/utils.ts

```diff
@@ -208,3 +210,23 @@ export const base64ToHex = (b64: string) => {
 export const hexToUint8Array = (hex: string) => {
   return Uint8Array.from(Buffer.from(hex, 'hex'));
 };
+
+/** Converts input to EIP55 format.
```

Comment:
Minor style nit but I think for the comments to be properly parsed the text should start on the line after `/**` when it's a multi-line comment block, you can find examples elsewhere if you're unclear.